xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -3627,7 +3627,7 @@ platform.linkchecker.livetable.date=Last Checked
 
 ### Dashboard Application Resources
 platform.dashboard.user.preferences=Dashboard preferences
-platform.dashboard.user.displayOnMainPage=Display my dashboard on the wiki home when I'm logged in (instead of the default dashboard)
+platform.dashboard.user.displayOnMainPage=Display my dashboard when I'm logged in (instead of the default dashboard)
```

Comment:
Well the reason I didn't like your version is because it's not the truth ;) You don't see the dashboard when you're logged in. You only see it when you go to the dashboard app home page... 

Ok if you want to mention it's replacing the default dashboard. I'd say:

"Replace the default dashboard by my custom dashboard"

WDYT?